### ISSUE TITLE
eslint warning or error information not clear when save the file and trigger recompile

### ISSUE THREAD

**Issue by hackftz** [fi3ework/vite-plugin-checker#83]
hello, I got a minor question and I think it is not the problem from vite, so I wanna to talk about this.

I write some code that can cause several error message, if I fix part of it and save file, the project reload and the remain error message overlay the old thing. Up to now it seems ok, but when I fix all of it, I expected all eslint error message can get clear but it still show the previous error message in terminal screen. It looks a bit of weired.

#### phenomenon

![image](https://user-images.githubusercontent.com/33424860/139372161-9c746d4f-b197-4c89-bd5d-8ee5d2a0e7e4.png)

![image](https://user-images.githubusercontent.com/33424860/139372132-697e4e13-7aaa-4413-94ec-ef7342748c80.png)

#### vite config

```ts
import checker from 'vite-plugin-checker';

// ...

return defineConfig({
    plugins: [
      reactRefresh(),
      checker({
        typescript: true,
        eslint: {
          files: ['./src', './config'],
          extensions: ['.ts', '.tsx', '.js', '.jsx'],
        },
      }),
    ],
  });
```

**Comment by fi3ework** [comment]
Sorry for the late response. I gave it a try but I can not reproduce this phenomenon. Could you provide a minimal repo or describe how to remain an unexpected error message in the terminal in more detail.

**Comment by fi3ework** [comment]
the issue should be fixed in 0.4.0. There's a little modification of the config of eslint. See [documentation](https://github.com/fi3ework/vite-plugin-checker#configeslint) and [example](https://github.com/fi3ework/vite-plugin-checker/blob/main/playground/eslint/vite.config.ts).
